index.html

```diff
@@ -1,57 +1,329 @@
+  <div class="pheading">
+  <h1>Tonic</h1>
+  </div>
+  <div class="date">
+  <ul>
+  <li>Canopy</li>
+  <li><img src="images/dot.png" alt=""></li>
+  <li>Back End Dev</li>
+
+  <li><img src="images/dot.png" alt=""></li>
+  <li>2015</li>
+
+  </ul>
+
+  </div>
+  <div class="summary">
+  <h2>A daily selection of privately personalized reads; no accounts or sign-ups required.</h2>
+  </div>
+
+  <div class="languages">
+
+
+  <ul>
+  <li>html</li>
+  <li>Css</li>
+  <li>javascript</li>
+
+  </ul>
+
+
+
+  </div>
+
+  <div class="see-project">
+  <button>See Project</button>
+  </div>
+
+  </div>
+
+
+  </section>
+  <br>
+  <br>
+  <br>
+  <section>
+
+  <div class="snapshot-container">
+
+  <div class="snapshot"><img src="images/Snapshoot Portfolio2.png" alt=""></div>
+  <div class="pheading">
+  <h1>Multi-Post Stories</h1>
+  </div>
+  <div class="date">
+  <ul>
+  <li>Canopy</li>
+  <li><img src="images/dot.png" alt=""></li>
+  <li>Back End Dev</li>
+
+  <li><img src="images/dot.png" alt=""></li>
+  <li>2015</li>
+
+  </ul>
+
+  </div>
+  <div class="summary">
+  <h2>A daily selection of privately personalized reads; no accounts or sign-ups required.</h2>
+  </div>
+
+  <div class="languages">
+
+
+  <ul>
+  <li>html</li>
+  <li>Css</li>
+  <li>javascript</li>
+
+  </ul>
+
+
+
+  </div>
+
+  <div class="see-project">
+  <button>See Project</button>
+  </div>
+
+  </div>
+
+
+  </section>
+
+  <br>
+  <br>
+  <br>
+  <section>
+
+  <div class="snapshot-container">
+
+  <div class="snapshot"><img src="images/Snapshoot Portfolio3.png" alt=""></div>
+  <div class="pheading">
+  <h1>Tonic</h1>
+  </div>
+  <div class="date">
+  <ul>
+  <li>Canopy</li>
+  <li><img src="images/dot.png" alt=""></li>
+  <li>Back End Dev</li>
+
+  <li><img src="images/dot.png" alt=""></li>
+  <li>2015</li>
+
+  </ul>
+
+  </div>
+  <div class="summary">
+  <h2>A daily selection of privately personalized reads; no accounts or sign-ups required.</h2>
+  </div>
+
+  <div class="languages">
+
+
+  <ul>
+  <li>html</li>
+  <li>Css</li>
+  <li>javascript</li>
+
+  </ul>
+
+
+
+  </div>
+
+  <div class="see-project">
+  <button>See Project</button>
+  </div>
+
+  </div>
+
+
+  </section>
+
+  <br>
+  <br>
+  <br>
+
+  <section>
+
+  <div class="snapshot-container">
+
+  <div class="snapshot"><img src="images/Snapshoot Portfolio4.png" alt=""></div>
+  <div class="pheading">
+  <h1>Multi-Post Stories</h1>
+  </div>
+  <div class="date">
+  <ul>
+  <li>Canopy</li>
+  <li><img src="images/dot.png" alt=""></li>
+  <li>Back End Dev</li>
+
+  <li><img src="images/dot.png" alt=""></li>
+  <li>2015</li>
+
+  </ul>
+
+  </div>
+  <div class="summary">
+  <h2>A daily selection of privately personalized reads; no accounts or sign-ups required.</h2>
+  </div>
+
+  <div class="languages">
+
+
+  <ul>
+  <li>html</li>
+  <li>Css</li>
+  <li>javascript</li>
+
+  </ul>
+
+
+
+  </div>
+
+  <div class="see-project">
+  <button>See Project</button>
+  </div>
+
+  </div>
+
+
+  </section>
```

Comment:
- [ ] You don't know to create a separate section for each project. They should all be inside one section and use `<article>` tag to wrap each of the projects and their respective information as this tag is used for repeated part inside a project.